src/Resources/contao/config/config.php

```diff
@@ -13,3 +13,8 @@
 $GLOBALS['TL_HOOKS']['sqlGetFromDca']['videoBundle'] = [\HeimrichHannot\VideoBundle\EventListener\SqlGetDataListener::class, 'onSqlGetFromDca'];
 
 $GLOBALS['TL_HOOKS']['initializeSystem']['huh_video'] = [\HeimrichHannot\VideoBundle\EventListener\InitializeSystemListener::class, '__invoke'];
+
+if ('BE' === TL_MODE) {
```

Comment:
Sowas kommt bitte in einen Hook und wird mit dem Container-Util abgefragt. Könntest den initializeSystemHook dafür verwenden.